slider/beatmap.py

```diff
@@ -1680,6 +1680,58 @@ def _resolve_stacking_old(self, hit_objects, ar, cs):
 
         return hit_objects
 
+    @lazyval
+    def _hit_object_times(self):
+        """a (sorted) list of hitobject time's, so they can be searched with
+        ``np.searchsorted``
+        """
+        return [hitobj.time for hitobj in self._hit_objects]
+
+    def closest_hitobject(self, t, side="left"):
+        """The hitobject closest in time to ``t``.
+
+        Parameters
+        ----------
+        t : :class:`datetime.timedelta`
```

Comment:
This shouldn't be a sphinx link, just `datetime.timedelta`. This will be a link anyway because of the `numpydoc` plugin.

Reply:
interesting, didn't know it auto-linked like that

Reply:
yeah, you can also set up mappings for linking to external projects like numpy and pandas if their docs are in sphinx.